mcp/src/test/java/io/modelcontextprotocol/server/transport/HttpServletSseServerTransportProviderIntegrationTests.java

```diff
@@ -44,7 +45,7 @@
 
 public class HttpServletSseServerTransportProviderIntegrationTests {
 
-	private static final int PORT = 8185;
+	private static final int PORT = 8189;
```

Comment:
Why change? Perhaps we need a simple utility to allocate a random port instead of relying on a constant? https://stackoverflow.com/questions/2675362/how-to-find-an-available-port

Reply:
This makes sense. Spring Boot used to have such utility but discarded it for some reason. 
Lets handle this in a dedicated issue/pr